js/constants/messages.js

```diff
@@ -111,8 +110,9 @@ const messages = {
   LAST_WINDOW_STATE: _,
   UNDO_CLOSED_WINDOW: _,
   // Menu rebuilding
-  REQUEST_WINDOW_STATE_FOR_MENU: _,
-  RESPONSE_WINDOW_STATE_FOR_MENU: _,
+  REQUEST_MENU_DATA_FOR_WINDOW: _,
+  RESPONSE_MENU_DATA_FOR_WINDOW: _,
+  UPDATE_MENU_BOOKMARKED_STATUS: _, /** @arg {Object} currently only has a boolean "bookmarked" */
```

Comment:
nit documentation bug, I think this arg is actually a boolean type and not an Object